Gemfile

```diff
@@ -3,6 +3,8 @@ source 'https://rubygems.org'
 # Specify your gem's dependencies in metasploit-model.gemspec
 gemspec
 
+gem 'metasploit-erd', git: 'https://github.com/adfoster-r7/metasploit-erd', branch: 'add-support-for-rails-7'
```

Comment:
https://github.com/rapid7/metasploit-erd/pull/18

Reply:
Gem is released this can be removed.